conferences/models.py

```diff
@@ -1,21 +1,30 @@
 from django.db import models
 from django.utils.translation import ugettext as _
+from django.core.exceptions import ValidationError
 
 
 class Place(models.Model):
-    place_name = models.CharField(
+    name = models.CharField(
         max_length=300
     )
-    place_url = models.CharField(
-        max_length=300
+    url = models.URLField(
+        max_length=300,
+        blank=True
     )
-    place_address = models.TextField()
+    address = models.TextField()
 
     def __str__(self):
-        return self.place_name
+        return self.name
+
+
+class ZosiaManager(models.Manager):
```

Comment:
A nie lepiej to przerzucić do innego pliku, np `managers.py` - wtedy plik z modelami będzie bardziej czytelny jeżeli powstanie dużo managerów i klas pobocznych.